x-pack/plugins/security_solution/server/lib/telemetry/task.ts

```diff
@@ -135,30 +136,32 @@ export class SecurityTelemetryTask {
         params: { version: this.config.version },
       });
     } catch (e) {
-      this.logger.error(`[task ${taskId}]: error scheduling task, received ${e.message}`);
+      this.logger.error('Error scheduling task', {
+        error: e.message,
+      } as LogMeta);
     }
   };
 
   public runTask = async (taskId: string, executionPeriod: TaskExecutionPeriod) => {
-    tlog(this.logger, `[task ${taskId}]: attempting to run`);
+    this.logger.l('Attempting to run', { taskId });
```

Comment:
Are these logging to the Kibana logger at INFO or higher?  I'd suggest using DEBUG if it's INFO, as we get sooo much noise in the Kibana logs for things like this.

It's always a good idea to debug log at the beginning and end of a task, so you at least know it's running.